packages/specs/pages/base-ledger/governance.mdx

```diff
@@ -128,28 +129,37 @@ pub enum ProposalType {
 
 - Can carry a wasm code to be executed in case the proposal passes
 - Allows both validators and delegators to vote
-- Requires 2/3 of the total voting power to succeed
+- Requires $\frac{2}{3}$ of the total voting power to succeed
 - Doesn't expect any memo attached to the votes
 
-`StewardProposal` is a proposal to elect a steward for _Public Goods Funding_:
+`StewardProposal` is a proposal to change the StewardSet for _Public Goods Funding_:
 
 - Doesn't carry any wasm code
 - Allows both validators and delegators to vote
-- Requires 1/3 of the total voting power to vote `Yay`
+- Requires $\frac{1}{3}$ of the total voting power to vote
+- Requires $\frac{1}{2}$ of the total votes to be `Yay`
+
+`StewardFundingProposal` is a proposal to *conduct* _Public Goods Funding_ **by a steward**:
+- Can only be submitted by PGF stewards
+- Allows both validators and delegators to vote
+- Passes by default unless $\frac{1}{3}$ of the total voting power votes on the proposal, AND out of the votes, at least $\frac{1}{2}$ are `Nay`
```

Comment:
Should we add a line about what happens when 2/3 vote and more than 2/3 of the votes are against? (steward is kicked)